public/plugins/ckeditor/skins/bootstrapck/scss/browser-specific/opera/dialog_opera.scss

```diff
@@ -0,0 +1,31 @@
+/*
+Copyright (c) 2003-2014, CKSource - Frederico Knabben. All rights reserved.
+For licensing, see LICENSE.md or http://ckeditor.com/license
+*/
+
+/*
+dialog_opera.css
+===============
+
+This file contains styles to used by all versions of Opera only.
+*/
+
+/* Base it on dialog.css, overriding it with styles defined in this file. */
+@import "../../dialog/dialog";
+
+/* Opera has problem with box-shadow and td with border-collapse: collapse */
+/* inset shadow is mis-aligned */
+.cke_dialog_footer
+{
+    display: block;
+    height: 38px;
+}
+
+.cke_ltr .cke_dialog_footer > *
+{
+    float:right;
+}
+.cke_rtl .cke_dialog_footer > *
```

Comment:
Selector `cke_rtl` should be written in hyphenated BEM (Block Element Modifier) format<br>Selector `cke_dialog_footer` should be written in hyphenated BEM (Block Element Modifier) format